tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Builders/OrderBuilder.cs

```diff
@@ -1,131 +1,172 @@
-﻿using System;
-using NHSD.BuyingCatalogue.Ordering.Domain;
+﻿﻿using System;
+ using System.Collections.Generic;
+ using NHSD.BuyingCatalogue.Ordering.Domain;
 
 namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Builders
 {
     internal sealed class OrderBuilder
     {
-        private readonly Order _order;
+        private string _orderId;
+        private string _orderDescription;
```

Comment:
For the record, I'm not a massive fan of changing our builders to this new structure, as I think it reads better the old way. I was going to bring it up last Fri, but then we postponed it. Up to you though

Reply:
This was reviewed during a coding standards meeting & I thought this was what we agreed. Creating the object in the constructor makes it difficult to test nullability. Take a Controller or a Service class for instance. If the object is created in the constructor you cannot pass null in as a parameter as that will throw an exception. This means a builder cannot be used in an argument null exception test, which defeats the purpose of the builder to encapsulate the creation of an object to one place so if the constructor changes we only have to change it in the builder & not in a set of test methods. Welcome to bring it up again if you can see advantages for the other approach.